### ISSUE TITLE
Standards visibility

### ISSUE THREAD

**Issue by JeremyFyke** [ECCC-CCCS/CCCS-data-standards#6]
Hosting this standards document via Github allows viewing 'privately', given collaborators have a github account.  Conversely, it allows full public viewing.  Is opening up publicly during final review stages appropriate, do you think @CarringtonPomeroy?

**Comment by CarringtonPomeroy** [comment]
@JeremyFyke Do we envision this being a public document in the end? I would imagine so? If so, then yes I think that would be appropriate.

Otherwise, I think those who would want to view it would either have a GitHub account or be more than capable of getting one. In cases where they don't then perhaps we could just circulate a word doc and incorporate comments for them.

**Comment by JeremyFyke** [comment]
Yes I'd envisage this being public eventually.  I'd tend to agree, that at this point it is OK for it to be public.

And good point, about the audience for this particular document being tech-savvy enough for the most part, to get a github account.  So if we keep it private, we can just direct folks to it, after first directing them to get a github account (if they don't already have).  But conversely, perhaps github is *not* the place to share draft/limited documents, if these audiences may not be tech-savvy by default.

**Comment by JeremyFyke** [comment]
BTW, I have now made this repo public, in case you wanted to share with anyone (Data Working Group, etc.)

**Comment by CarringtonPomeroy** [comment]
Agree that it will depend on audience
